.github/workflows/percy-snapshots.yml

```diff
@@ -13,3 +13,4 @@ jobs:
       percy_script: npx percy storybook ./storybook-static
     secrets:
       PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+      REACT_APP_MAPBOX_API_KEY: ${{ secrets.REACT_APP_MAPBOX_API_KEY }}
```

Comment:
I don't think the reusable workflow for the Percy snapshots expects a secret with this name, so you might have to update that [workflow](https://github.com/yext/slapshot-reusable-workflows/blob/main/.github/workflows/percy_snapshots.yml) too

Reply:
@nmanu1 you're right the build failed `Invalid secret, REACT_APP_MAPBOX_API_KEY is not defined in the referenced workflow`. Do I just need to merge that into v1 (like the other prs) or is there ui stuff to do in GitHub as well?

Reply:
no additional setup is needed -- just need the pr with the changes to merge to v1 branch (and main branch) in this [repo](https://github.com/yext/slapshot-reusable-workflows)

Reply:
talked offline